docs/guide/docs/nlu/entity-extraction.md

```diff
@@ -1 +1,193 @@
+    source: '5 lbs', // text from which the entity was extracted
+    start: 4, // beginning character index in original input
+    end: 9, // end character index in original input
+  },
+  data: {
+    value : 5,
+    unit: 'pound',
+    extras: {}
+  }
+}
+```
+
+> **Note**: Confidence will always be one due to the rule-based implementation of Duckling
+
+## Custom Entities
+Botpress provides two types of custom entities: [pattern](#pattern-extraction) and [list](#list-extraction) entities. To define a custom entity, go to the **Entity section** of the NLU Module interface accessible from the Botpress studio sidebar. From there, you can define your custom entities which will be available for any input message treated by your chatbot. Go ahead and click on **create new entity**
+
+[Creating a new entity](../assets/nlu-create-entity.png)
+
+### Pattern extraction
+Pattern or Regular Expression Extraction allows you to extract information presented in a format that can be described using Regular Expression (RegEx). Once you've created a pattern entity, Botpress Native NLU will perform a regex extraction on each incoming message and add it to `event.nlu.entities`.
+
+#### Example :
+
+Given a Pattern Entity definition with `[A-Z]{3}-[0-9]{4}-[A-Z]{3}` as pattern:
+
+![create slot](assets/nlu-pattern-entity.png)
+
+Extraction will go like this:
+
+|           User said           | Type  |     Value      |
+| :---------------------------: | :---: | :------------: |
+| _"Find product BHZ-1234-UYT"_ | "SKU" | "BHZ-1234-UYT" |
+
+```js
+{ name: 'SKU',
+  type: 'pattern',
+  meta:
+   { confidence: 1,
+     provider: 'native',
+     source: 'BHZ-1234-UYT',
+     start: 13,
+     end: 25,
+     raw: {} },
+  data: {
+    extras: {},
+    value: 'BHZ-1234-UYT',
+    unit: 'string'
+    }
+}
+```
+
+### List extraction
+
+List extraction behaves similarly to pattern extraction. However, you'll be able to add different **occurrences** of your entity with corresponding synonyms.
+
+Let's take **Airport Codes** as an example:
+
+![create slot](assets/nlu-list-entity.png)
+
+Extraction will go like this:
+
+|              User said               |      Type       |     Value      |
+| :----------------------------------: | :-------------: | :------------: |
+| _"Find a flight from SFO to Mumbai"_ | "Airport Codes" | ["SFO", "BOM"] |
+
+```js
+;[
+  {
+    name: 'Airport Codes',
+    type: 'list',
+    meta: {
+      confidence: 1,
+      provider: 'native',
+      source: 'SFO',
+      start: 19,
+      end: 22,
+      raw: {}
+    },
+    data: {
+      extras: {},
+      value: 'SFO',
+      unit: 'string'
+    }
+  },
+  {
+    name: 'Airport Codes',
+    type: 'list',
+    meta: {
+      confidence: 1,
+      provider: 'native',
+      source: 'Mumbai',
+      start: 26,
+      end: 32,
+      raw: {}
+    },
+    data: {
+      extras: {},
+      value: 'BOM',
+      unit: 'string'
+    }
+  }
+]
+```
+
+## Placeholder extraction (experimental)
```

Comment:
Can you remove experimental here. It's not experimental it's just a hard task with a few training examples